### ISSUE TITLE
Wiki examples become outdated very often

### ISSUE THREAD

**Issue by Kakadu** [ocsigen/tutorial#20]
Right decision seems to enter text in .eliom file and use a tool which generates HTML from it. As a result we should get something like [software foundations](http://www.cis.upenn.edu/~bcpierce/sf/current/Logic.html). Using that tool will solve the problem of non compilable wiki examples.

@Drup , can you +1?

**Comment by agarwal** [comment]
FYI, we have a new project [oloop](https://github.com/agarwal/oloop) started to address similar issues for regular OCaml code. The hope is to make it easier to write ocaml.org style tutorials, blog posts, books, etc with embedded OCaml code. I'm not sure any of it can be reused for Ocsigen based tutorials, but just wanted to point it out.

**Comment by Drup** [comment]
Yes, I agree. I'm not sure wikidoc can do that and I hope we will get read of it and migrate to codoc at some point anyway. There are other rather nice tools (I personally like [http://mjambon.com/caml2html.html](caml2html)).

@agarwal oloop doesn't work, we want to compile our files, executing in a toploop ist not enough.
